game-core/src/main/java/games/strategy/engine/pbem/TripleAForumPoster.java

```diff
@@ -90,12 +92,8 @@ private static void deleteToken(final CloseableHttpClient client, final int user
       throws IOException {
     final HttpDelete httpDelete = new HttpDelete(tripleAForumURL + "/api/v2/users/" + userId + "/tokens/" + token);
     addTokenHeader(httpDelete, token);
-    try (CloseableHttpResponse response = client.execute(httpDelete)) {
-      final int code = response.getStatusLine().getStatusCode();
-      if (code != HttpURLConnection.HTTP_OK) {
-        // Cleanup failed, this is not a severe issue
-        ClientLogger.logQuietly("Failed to delete API Token after posting, server responded with error code " + code);
-      }
+    try (CloseableHttpResponse ignored = client.execute(httpDelete)) {
```

Comment:
Without any content, the response could immeaditely be closed instead of using a try with resource block

Reply:
That would mean the behavior of `client.execute` depends on calls in the next few lines, notably: ` response.getStatusLine().getStatusCode()`, that would be very surprising. Is that really the case? Is that a known problem for this http client? I'm a bit surprised this kind of gotchya would be there, it is quite the landmine.

Reply:
? I'm not sure what you want to say here.
I was only observing that the try with resource block is now empty, and the resource could immediately get closed directly instead of assigning it to a variable first.

Reply:
Gotchya - I actually did the same refactor at first   :)

eg:
```
client.execute(httpDelete).close();
```

But, with the above the `close()` is no longer in a finally block. That would be a functional change of course if `execute` throws an exception, we'll never get that close called.

Reply:
Ah ok, makes sense, although unlikely.

Reply:
Although in case an exception occurs. A Closeable Response won't be created in the first place, so meh